src/solver/model_getters.jl

```diff
@@ -36,28 +36,26 @@ end
 
 function get_hyperdiffusion_model(parsed_args, spaces, ::Type{FT}) where {FT}
     hyperdiff_name = parsed_args["hyperdiff"]
-    h_space = Spaces.horizontal_space(spaces.center_space)
-    h_length_scale = Spaces.node_horizontal_length_scale(h_space) # mean nodal distance
-    κ₄_vorticity = FT(parsed_args["kappa_4_vorticity"]) * h_length_scale^3
-    κ₄_scalar = FT(parsed_args["kappa_4_scalar"]) * h_length_scale^3
+    ν₄_vorticity = FT(parsed_args["kappa_4_vorticity"])
+    ν₄_scalar = FT(parsed_args["kappa_4_scalar"])
```

Comment:
may want to rename these arguments?

Reply:
how about `kappa_4_vorticity_coeff` and `kappa_4_scalar_coeff`, and `κ₄_vorticity_coeff` and `κ₄_scalar_coeff` instead of `ν₄`? I'm also ok with renaming all `κ₄` to `ν₄` (it may be even better).